PCGen-base/code/src/java/pcgen/base/util/TripleKeyMap.java

```diff
@@ -297,11 +285,8 @@ public V remove(K1 key1, K2 key2, K3 key3)
 	public Set<K3> getTertiaryKeySet(K1 key1, K2 key2)
 	{
 		Map<K3, V> localMap = map.get(key1, key2);
-		if (localMap == null)
-		{
-			return Collections.emptySet();
-		}
-		return new HashSet<>(localMap.keySet());
+		return (localMap == null) ? Collections.emptySet()
```

Comment:
ditto on nullToEmpty (then it can become new `HashSet<>(nullToEmpty(localMap))`)